nms/app/packages/magmalte/app/views/subscriber/SubscriberStateTable.js

```diff
@@ -0,0 +1,268 @@
+  service: string,
+  currentUsage: string,
+  dailyAvg: string,
+  lastReportedTime: Date | string,
+};
+
+export default function SubscriberStateTable(props: WithAlert & TableProps) {
+  const {history, relativeUrl} = useRouter();
+  const [currRow, setCurrRow] = useState<SubscriberRowType>({});
+  const enqueueSnackbar = useEnqueueSnackbar();
+  const ctx = useContext(SubscriberContext);
+  const state = ctx;
+  const networkCtx = useContext(NetworkContext);
+  // $FlowIgnore
+  const subscriberMap: {[string]: subscriber} = state.state;
+  // $FlowIgnore
+  const sessionState: {[string]: subscriber_state} = state.sessionState;
+  const subscriberMetrics = ctx.metrics;
+  const [jsonDialog, setJsonDialog] = useState(false);
+  const tableColumns = [
+    ...props.columns,
+    {
+      title: 'Active Sessions',
+      field: 'activeSessions',
+      width: 175,
+    },
+    {
+      title: 'Active APNs',
+      field: 'activeApns',
+    },
+    {
+      title: 'Session IP Address',
+      field: 'ipAddress',
+    },
+  ];
+
+  const subscribersIds = Object.keys(sessionState);
+  type SubscriberSessionRowType = {
+    apnName: string,
+    sessionId: string,
+    ipAddr: string,
+    state: string,
+    activeDuration: string,
+    activePolicies: Array<string>,
+  };
+
+  const tableData: Array<SubscriberRowType> = subscribersIds.map(
+    (imsi: string) => {
+      const subscriberInfo = subscriberMap[imsi] || {};
+      const metrics = subscriberMetrics?.[`${imsi}`];
+      const subscriber = sessionState[imsi]?.subscriber_state || {};
+      const ipAddresses = [];
+      const activeApns = [];
+      let activeSessions = 0;
+      Object.keys(subscriber || {}).forEach(apn => {
+        subscriber[apn].forEach(session => {
+          if (session.lifecycle_state === 'SESSION_ACTIVE') {
+            ipAddresses.push(session?.ipv4);
+            activeSessions++;
+          }
+        });
+        activeApns.push(apn);
+      });
+      return {
+        name: subscriberInfo.name ?? imsi,
+        imsi: imsi,
+        service: subscriberInfo.lte?.state || '',
+        currentUsage: metrics?.currentUsage ?? '0',
+        activeApns: activeApns.length > 0 ? activeApns.join() : '-',
+        activeSessions: activeSessions,
+        ipAddress: ipAddresses.length > 0 ? ipAddresses.join() : '-',
+        dailyAvg: metrics?.dailyAvg ?? '0',
+        lastReportedTime:
+          subscriberInfo.monitoring?.icmp?.last_reported_time === 0
+            ? new Date(subscriberInfo.monitoring?.icmp?.last_reported_time)
+            : '-',
+      };
+    },
+  );
+
+  const onClose = () => setJsonDialog(false);
+  return (
+    <>
+      <div>
+        <JsonDialog open={jsonDialog} onClose={onClose} imsi={currRow.imsi} />
+        <ActionTable
+          data={tableData}
+          columns={tableColumns}
+          handleCurrRow={(row: SubscriberRowType) => setCurrRow(row)}
+          menuItems={
+            networkCtx.networkType === FEG_LTE
```

Comment:
You can remove this FEG_LTE check in this table and keep only the View JSON option in this table. Meanwhile, the View, Edit and Remove options can be present in the Subscriber table.